src/contract_wrappers/exchange_wrapper.ts

```diff
@@ -714,7 +714,7 @@ export class ExchangeWrapper extends ContractWrapper {
         await assert.isUserAddressAvailableAsync(this._web3Wrapper);
         const exchangeInstance = await this._getExchangeContractAsync();
         const isRoundingError = await exchangeInstance.isRoundingError.call(
-            demoninator, numerator, makerTokenAmount,
+            numerator, demoninator, makerTokenAmount,
```

Comment:
No. We just did it wrong before.
I checked the contracts and it's: `function isRoundingError(uint numerator, uint denominator, uint target)`